src/Analyzers/CSharp/Tests/UseImplicitlyTypedLambdaExpression/UseImplicitlyTypedLambdaExpressionTests.cs

```diff
@@ -18,7 +19,7 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseImplicitlyTypedLambd
 [Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitObjectCreation)]
 public sealed class UseImplicitlyTypedLambdaExpressionTests
 {
-    private static readonly LanguageVersion CSharp14 = LanguageVersion.Preview;
+    private static readonly LanguageVersion CSharp14 = LanguageVersionExtensions.CSharpNext;
```

Comment:
we use CSharpNext as there's a process where we look for this when adding the actual CSharp14 LangVersion before shpping that release.